bchangg/on-air-device
# Add main and boot scripts. Add instruction for flashing firmwares and…

# Description, Motivation, and Context

- Add main and boot script for esp32 controller to connect to MQTT broker
- Add instruction to flash the firmware, also load the main and boot scripts with Arduino Micropython IDE


# Replication Steps / How to test
See Readme.md for details
- Flash firmware
- Upload scripts to device
- Send 'on' or 'off' message to 'light' topic
- profit
# Relevant Links (Stack Overflow, JIRA, RFC, etc)
How to flash esp32 firmware: https://learn.sparkfun.com/tutorials/how-to-load-micropython-on-a-microcontroller-board/esp32-thing
How to connect to MQTT broker with micropython: https://randomnerdtutorials.com/micropython-mqtt-esp32-esp8266/ 
Esp32 micropython firmware: https://micropython.org/download/esp32/
Arduino Micropython IDE: https://labs.arduino.cc/en/labs/micropython?_gl=1*1lofvmf*_ga*MjAwMzU5Nzc0NS4xNjc5MDk3OTc2*_ga_NEXN8H46L5*MTY3OTExNzY4My4zLjAuMTY3OTExNzY4My4wLjAuMA..
